sdk/core/azure-core/src/cryptography/md5.cpp

```diff
@@ -194,7 +194,6 @@ class Md5OpenSSL final : public Azure::Core::Cryptography::Hash {
     }
     if (1 != EVP_DigestInit_ex(m_context, EVP_md5(), NULL))
     {
-      EVP_MD_CTX_free(m_context);
```

Comment:
I understand your point, but teasing out these individual changes from this PR would be extremely difficult to do at this point, and would add several days of work on top of an already 1 week late feature (I'd have to extract all the changes, undo them from this PR (to avoid merge conflicts), commit them in another branch (a delay of 3-4 days because of code review delays), then integrate them back into my branch before I can get the CI pipeline working again.

That's a lot of work and I'm really not sure it's worth the effort at this point.

Reply:
One addendum: This PR is going into feature/websockets, *NOT* main.

Pulling changes from main to feature/websockets is quite challenging and requires a separate PR from main to feature/websockets. 

Putting these changes into main adds a number of additional because there needs to be a separate PR to move the changes from main into feature/websockets.

I tried merging main into this PR and it made every change from main into a part of this PR. That's a nightmare to manage. Not impossible, but a huge amount of work.